jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -696,8 +688,34 @@
         # Increase qps limits in scheduler and controller manager.
         export CONTROLLER_MANAGER_TEST_ARGS="--kube-api-qps=100 --kube-api-burst=100"
         export SCHEDULER_TEST_ARGS="--kube-api-qps=100 --kube-api-burst=100"
-        # Increase throughput in Load test.
-        export LOAD_TEST_THROUGHPUT=50
-        export KUBE_NODE_OS_DISTRIBUTION="debian"
+
+    suffix:
+        - 'gce-enormous-cluster':  # kubernetes-e2e-gce-enormous-cluster
+            description: 'Starts and deletes empty 2000 node cluster and runs scalability tests on it. Does allow few Nodes to fail during startup.'
+            timeout: 420
```

Comment:
Since we are still using 20 QPS, 420 is a risky - tests take ~7 hours in that case. So I would leave it at 1440 for now.